core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -234,6 +235,19 @@
 
 {% block content %}
   <div ng-controller="ExplorationEditor" ng-cloak>
+    <script type="text/ng-template" id="modals/shareExplorationAfterPublish">
+      <div class="modal-header">
+        <h3>Share Exploration</h3>
```

Comment:
For now, let's go with the following as the contents of the modal. We can change it later if necessary, but it seems like it's worth getting the PR in.

**Header:** Exploration Published

**Body:**

Congratulations! Your exploration was successfully published.

(new paragraph, in bold text) Share your exploration via: